tests/admin-hotel-tests/AdminHotelTest.php

```diff
@@ -0,0 +1,317 @@
+<?php
+
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use HotelBooking\AdminHotel;
+
+class AdminHotelTest extends TestCase
```

Comment:
you're testing for `AdminHotelController` but the class name is `AdminHotelTest`